src/SFML/Network/Packet.cpp

```diff
@@ -381,7 +381,7 @@ Packet& Packet::operator>>(String& data)
         {
             std::uint32_t character = 0;
             *this >> character;
-            data += character;
+            data += static_cast<char32_t>(character);
```

Comment:
Is there no `operator>>`  overload for `char32_t`?

Reply:
I think the `std::uint32_t` stream operations on `sf::Packet` need to be replaced with `char32_t` overloads.

Reply:
Makes sense. They could probably reuse the `uint32_t` implementation internally.

Reply:
On second thought, all those integer stream operations on `sf::Packet` exist for all integer types so we shouldn't remove the `std::uint32_t` overloads. We could add a `char32_t` but that seems a bit weird since no other character type gets its own overloads. _Pointers_ to character types are included but not character types by value. I'm not really sure what is the most proper solution is here.

Reply:
@eXpl0it3r Any API design preferences here? Do we add a convenience for `char32_t` serialization in spite of no other character types getting the same treatment? Do we add overloads for `char`, `char16_t`, and `char32_t`?